vterm.el

```diff
@@ -206,6 +206,8 @@ and background color are used for bright color. "
   "Delay for refreshing the buffer after receiving updates from libvterm.
 Improves performance when receiving large bursts of data.
 If nil, never delay")
+(defvar vterm-copy-map  (make-sparse-keymap))
+(defvar vterm-raw-map  (make-sparse-keymap))
```

Comment:
Please remove those two maps and instead use `(use-local-keymap nil)` to disable `vterm-mode-map` and use (use-local-keymap vterm-mode-map)` to activate it again.

Reply:
I removed  `term-raw-map` and leave `vterm-copy-map` as the keymap for `vterm-mode`